bin/dependency_validator.dart

```diff
@@ -53,6 +54,10 @@ final ArgParser argParser = new ArgParser()
     help: 'Comma-delimited list of packages to ignore from validation.',
     splitCommas: true,
   )
+  ..addFlag(ignoreCommonBinariesArg,
```

Comment:
Looks like the readme has the usage for this util, can we update that with this flag?

Reply:
Oops, I missed that. Sure can!